cmd/builder/internal/builder/main_test.go

```diff
@@ -113,6 +113,7 @@ var replaceModules = []string{
 func newTestConfig(tb testing.TB) *Config {
 	cfg, err := NewDefaultConfig()
 	require.NoError(tb, err)
+	cfg.Distribution.Name = "test_distribution"
```

Comment:
Currently, the integration tests are running with a blank name field, which means the args end up being `build -o  'ldflags=-s -w' ...`. So it's building a binary with the name 'ldflags=-s -w' instead of applying the ldflags. It didn't break any tests but is probably not intentional.